### ISSUE TITLE
Use of sections instead of divs.

### ISSUE THREAD

**Issue by farishkash** [acasas1826/dasmoto#2]
Sections have specific meaning and purpose in html.  In most cases you should be using a div.  Especially if you want your site to make sense to screen readers for the visually impaired.

Some notes from the html 5 spec with regards to sections.

> Examples of sections would be chapters, the various tabbed pages in a tabbed dialog box, or the numbered sections of a thesis.
> 
> 
> The section element is not a generic container element. When an element is needed only for styling purposes or as a convenience for scripting, authors are encouraged to use the div element instead. A general rule is that the section element is appropriate only if the element's contents would be listed explicitly in the document's outline.
>

**Comment by acasas1826** [comment]
I will follow your advice! Thanks!!
